enhancements/dns/coredns-egressfirewall-integration.md

```diff
@@ -128,20 +128,23 @@ type DNSNameResolver struct {
 	Status DNSNameResolverStatus `json:"status,omitempty"`
 }
 
+// DNSName is used for validation of a DNS name.
+// +kubebuilder:validation:Pattern=`^(\*\.)?([a-z0-9]([-a-z0-9]{0,61}[a-z0-9])?\.){2,}$`
+// +kubebuilder:validation:MaxLength=254
```

Comment:
validation changes are coming from the api PR feedback right?

Reply:
> validation changes are coming from the api PR feedback right?

Yes.